src/components/EllipsisLoader/index.js

```diff
@@ -1,37 +1,17 @@
 /** @format */
 
 import React from 'react';
-import PropTypes from 'prop-types';
 import './scss/Styles.scss';
 
-const EllipsisLoader = ({ dotColor }) => (
+const EllipsisLoader = () => (
```

Comment:
since we are changing things.... let me suggest:
1. use a 'baseClass' prop that is used in each className
2. simplify the ellipsis/dot class name to something like: `${baseClass}__dot`

Reply:
I really like this idea.

Reply:
done